core-web/libs/sdk/uve/src/lib/editor/public.ts

```diff
@@ -0,0 +1,76 @@
+import { DotCMSReorderMenuConfig, DotCMSUVEMessage } from '../types/editor/internal';
+import { Contentlet, DotCMSUVEAction } from '../types/editor/public';
+import { DotCMSInlineEditingPayload, DotCMSInlineEditingType } from '../types/events/public';
+
+/**
+ * Post message to dotcms page editor
+ *
+ * @export
+ * @template T
+ * @param {DotCMSUVEMessage<T>} message
+ */
+export function sendMessageToEditor<T = unknown>(message: DotCMSUVEMessage<T>) {
+    window.parent.postMessage(message, '*');
+}
```

Comment:
I think in some convo someone mention that `sendMessageToEditor` was not a good name.

Reply:
We can try `sendEditorcommand()` since our `actions` are commands to the editor